ouroboros-network/sim-tests-lib/Test/Ouroboros/Network/PeerSelection.hs

```diff
@@ -1726,7 +1726,7 @@ prop_governor_target_known_5_no_shrink_below env =
               --
               -- By subtracting a sum of `y` and `y'` we also do not account
               -- forgetting big ledger peers.
-              (\(x,y) (x',y') -> x Set.\\ x' Set.\\ (y <> y'))
+              (\(x,y) (x',y') -> x Set.\\ x' Set.\\ y Set.\\ y')
```

Comment:
Why this change? It seems the comment was aware of that sum

Reply:
there's no difference, isn't it?

Reply:
Right, I think that's right 😄